app/models/agents/trello_move_cards_agent.rb

```diff
@@ -0,0 +1,95 @@
+require 'trello'
+
+module Agents
+  class TrelloMoveCardsAgent < Agent
+    no_bulk_receive!
+    cannot_be_scheduled!
+    cannot_create_events!
+    gem_dependency_check { defined?(Trello) }
+
+    description <<-MD
+      The Trello Move Cards agent allows you to move card(s) from one list to another. Once the agent receives an event it will execute. The format/content of the event does not matter.
+
+      To use the Trello Move Cards agent you will first need to sign into your Trello account.
+      Once you have signed in go to the following url [https://trello.com/app-key](https://trello.com/app-key) and copy the key.
+      This will be used as the agent `public_key`.
+
+      Next on the same page click on the `Token` link and copy the token displayed. This will be used as the agent `member_token`.
```

Comment:
Yes, I think that'd be better.  We have a concept of Services that authenticate with oAuth.